atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraConstants.java

```diff
@@ -50,9 +50,9 @@
     static final String DEFAULT_RACK = "rack1";
     static final String SIMPLE_RF_TEST_KEYSPACE = "__simple_rf_test_keyspace__";
     static final String REPLICATION_FACTOR_OPTION = "replication_factor";
-    // 4 days; Hinted-Handoffs MUST expire well within this period for delete correctness
+    // 1 hour; Hinted-Handoffs MUST expire well within this period for delete correctness
     // (I believe we will be expiring hints in half this period)
```

Comment:
Is this line still true?  Is the expiration time for hints defined to be half the GC grace time (in which case we should make this comment more definitive) or is it separately configured (in which it is now out of date with this config)?